qcfractal/tests/test_collections.py

```diff
@@ -732,10 +745,56 @@ def test_compute_reactiondataset_keywords(fractal_compute_server):
     assert kw.values["scf_type"] == "df"
 
 
+def live_fractal_or_skip():
+    """Ensure Fractal live connection can be made"""
+    import requests
```

Comment:
Fine to have requests as a top level import as it is imported into the project anyway.